test/agentchat/test_compressible_agent.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+import sys
+import autogen
+from test_assistant_agent import KEY_LOC, OAI_CONFIG_LIST
+
+try:
+    from autogen.agentchat.contrib.compressible_agent import (
+        CompressibleAgent,
+    )
+
+    skip_test = False
+except ImportError:
+    skip_test = True
```

Comment:
instead of setting skip_test to True and silently skipping the test, it might be better to log or print an informative message for clarity.

Reply:
This was copied from another test and is not necessary here, thank you for pointing it out.